diesel/src/query_dsl/having_dsl.rs

```diff
@@ -0,0 +1,29 @@
+use crate::dsl::Having;
+use crate::query_source::*;
+
+/// The `having` method
+///
+/// This trait should not be relied on directly by most apps. Its behavior is
+/// provided by [`QueryDsl`]. However, you may need a where clause on this trait
+/// to call `having` from generic code.
+///
+/// [`QueryDsl`]: ../trait.QueryDsl.html
```

Comment:
We can use the intra-doc link here:

```suggestion
/// [`QueryDsl`]: crate::QueryDsl
```